config/serverless.yml

```diff
@@ -137,10 +137,12 @@ xpack.alerting.rules.maxScheduledPerMinute: 400
 xpack.actions.run.maxAttempts: 10
 xpack.actions.queued.max: 10000
 
-# Disables ESQL in advanced settings (hides it from the UI)
 uiSettings:
   overrides:
+    # Disables ESQL in advanced settings (hides it from the UI)
     enableESQL: true
+    # Disables `Defer loading panels below "the fold"`
+    labs:dashboard:deferBelowFold: false
```

Comment:
Just curious, how does this affect performance of dashboards?

Reply:
Can someone from @elastic/kibana-presentation answer this please?

Reply:
I don't think there are any metrics captured to quantize the effect of this setting. @thomasneirynck has been capturing dashboard load metrics. Thomas, do you have any data about effects of "Defer loading panels below "the fold" setting on dashboard load performance?

The best practices for dashboards are to avoid large dashboards that need scrolling and instead use links panel to create smaller dashboards

Reply:
@nreese no, we don't track this setting metric in any of the performance journeys